pkg/operator/operator.go

```diff
@@ -85,20 +93,22 @@ func init() {
 }
 
 // NewOperator returns a new Dapr Operator.
-func NewOperator(opts Options) Operator {
+func NewOperator(ctx context.Context, opts Options) (Operator, error) {
 	conf, err := ctrl.GetConfig()
 	if err != nil {
-		log.Fatalf("Unable to get controller runtime configuration, err: %s", err)
+		return nil, fmt.Errorf("Unable to get controller runtime configuration, err: %s", err)
 	}
 	mgr, err := ctrl.NewManager(conf, ctrl.Options{
-		Scheme:             scheme,
-		MetricsBindAddress: "0",
-		LeaderElection:     opts.LeaderElection,
-		LeaderElectionID:   "operator.dapr.io",
-		Namespace:          opts.WatchNamespace,
+		Scheme:                 scheme,
+		Port:                   19443,
```

Comment:
Why is the port hardcoded?

Reply:
The webhook port was already hardcoded.

https://github.com/dapr/dapr/pull/5859/files#diff-eb049014aed7f180e4ec0ec97c303124cc17ca08d97f9520bc0c4637a6fd1fdeL34